Sources/EduIDExpansion/Flows/Activity/ActivityNavigationDelegate.swift

```diff
@@ -0,0 +1,5 @@
+import UIKit
+
+protocol ActivityNavigationDelegate: AnyObject {
+    func dismissActivityFlow(sender: AnyObject)
+}
```

Comment:
```swift
protocol ActivityViewControllerDelegate: AnyObject {
     func activityViewControllerDidDismissActivityFlow(viewController: ActivityViewController)
 }
```

Reply:
this contradicts that the sender can be removed as a child coordinator

Reply:
I did this

Reply:
Are there still unpushed changes?